### ISSUE TITLE
Error when importing into a electron-vue app

### ISSUE THREAD

**Issue by devszr** [israelss/vue-worker#11]
Hi,

I'm trying to use VueWorker in an app created with electron-vue, [https://github.com/SimulatedGREG/electron-vue]. 

When importing VueWorker, I get the following error,
`Uncaught SyntaxError: Unexpected token import
    at createScript (vm.js:74)
    at Object.runInThisContext (vm.js:116)
    at Module._compile (module.js:533)
    at Object.Module._extensions..js (module.js:580)
    at Module.load (module.js:503)
    at tryModuleLoad (module.js:466)
    at Function.Module._load (module.js:458)
    at Module.require (module.js:513)
    at require (internal/module.js:11)
    at eval (external "vue-worker"?b6d5:1)`

I don't know if this is a problem with electron-vue or with VueWorker, thought I'll ask here first.
The error is reported in \node_modules\vue-worker\index.js:1.

Steps:

Create an empty project with `vue init simulatedgreg/electron-vue my-project`

Then npm install vue-worker and import it in the main.js created by electron-vue.

As soon as you start the application, you'll see the error.

Any ideas ?

Thanks.

I posted an issue on electron-vue as well, SimulatedGREG/electron-vue/issues/562

**Comment by mistricky** [comment]
@devszr Have you solved the problem? I have a same problem.

**Comment by devszr** [comment]
@HaoDaWang Nope. I ended up not using vue-worker.

**Comment by sjeon87** [comment]
@devszr Did you find an alternative?

**Comment by devszr** [comment]
@gsongsong  Not really. I used Node's child_process module directly without using webworkers since my use case was within an Electron App.

**Comment by sjeon87** [comment]
@devszr Sad. I adopted suggestion described in https://stackoverflow.com/a/40251412/3278063.

A *worker* window is hidden (`{show: false}`) and routed to `#/worker`, which renders page with just an empty template and `<script></script>`, the real part.

I am not sure this approach has limitations, but so far it looks goot to me. Bothersome part is on main process bridging two (or more) renderer processes.

**Comment by SpotlightPL** [comment]
@devszr you can simply whitelist vue-worker inside rendered config.
https://github.com/SimulatedGREG/electron-vue/issues/284
